src/porepy/models/geometry.py

```diff
@@ -203,6 +203,13 @@ def interfaces_to_subdomains(
                     subdomains.append(sd)
         return self.mdg.sort_subdomains(subdomains)
 
+    def subdomains_to_boundary_grids(
+        self, subdomains: Sequence[pp.Grid]
+    ) -> Sequence[pp.BoundaryGrid]:
+        """TODO"""
+        boundary_grids = [self.subdomain_to_boundary_grid(sd) for sd in subdomains]
```

Comment:
I'm slightly confused. We have `mdg.subdomain_to_interfaces` because one subdomain may have several associated interfaces, and also we have a function `subdomains_to_interfaces`, which is a shortcut that does the same, but for several subdomains. In the same logic, we have `mdg.subdomain_to_boundary_grid`, singular because only one BG is associated with the subdomain, and the shortcut function `subdomains_to_boundary_grids`. Do you propose to keep only the plural ones for both interfaces and boundary grids?